app/javascript/components/recipes/new/add-button.vue

```diff
@@ -0,0 +1,18 @@
+ <template> 
```

Comment:
⚠️ **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected indentation of 0 spaces but found 1 space                                vue/html-indent